spec/e2e/help_spec.sh

```diff
@@ -15,7 +15,8 @@ Available Commands:
   test        Execute Rego test cases
 
 Flags:
-  -h, --help   help for snyk-iac-rules
+  -h, --help      help for snyk-iac-rules
+  -v, --version   version for snyk-iac-rules
```

Comment:
It was fiddly to change this test and I'm not sure what it really guards against - it's just double-entry book-keeping. No harm done I guess, but perhaps we'd consider removing this particular test (outside the scope of this PR)